basic_flutter/lib/main.dart

```diff
@@ -10,30 +12,66 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      home: const MyHomePage(
         title: "Employee App",
-        home: const MyHomePage(),
-        theme: ThemeData(
-            primaryColor: Colors.blue.shade900,
-            splashColor: Colors.blue,
-            textTheme: ThemeData.light().textTheme.copyWith(
-                headline6: const TextStyle(
-                    fontFamily: "Raleway-Bold", fontSize: 20))));
+      ),
+      theme: ThemeData(
+        primaryColor: Colors.blue.shade900,
+        splashColor: Colors.blue,
+        textTheme: ThemeData.light().textTheme.copyWith(
+              headline6:
+                  const TextStyle(fontFamily: "Raleway-Bold", fontSize: 20),
+            ),
+      ),
+    );
   }
 }
 
 class MyHomePage extends StatefulWidget {
-  const MyHomePage({Key? key}) : super(key: key);
+  const MyHomePage({Key? key, required this.title}) : super(key: key);
+  final String title;
 
   @override
-  _MyHomePageState createState() => _MyHomePageState();
+  State<MyHomePage> createState() => _MyHomePageState();
 }
 
 class _MyHomePageState extends State<MyHomePage> {
+  @override
+  void initState() {
+    super.initState();
+    Timer(
+      const Duration(seconds: 3),
+      () {
+        Navigator.of(context).pushReplacement(
+          MaterialPageRoute(
+            builder: (_) => const DummyPage(),
+          ),
+        );
+      },
+    );
+  }
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        appBar: AppBar(title: const Text("Employee App")),
-        body: const Center(
-            child: Text("Hello World", style: TextStyle(fontSize: 30))));
+      backgroundColor: Colors.blue.shade900,
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: [
+            Image.network(
+              'https://raw.githubusercontent.com/devonfw-sample/basic-flutter/45cadd7afb1c0d891c380cd76f068c06426b50b6/basic_flutter/assets/logo.PNG',
```

Comment:
is it necessary to load the image from the internet or is it possible to just have the image in the app?

Reply:
We can also load the image from the app.